lib/rasn1/model.rb

```diff
@@ -106,27 +106,27 @@ def inherited(klass)
       end
 
       def define_type_accel_base(accel_name, klass)
-        singleton_class.class_eval(
-          "def #{accel_name}(name, options={})\n" \
-          "  options[:name] = name\n" \
-          "  proc = proc do |opts|\n" \
-          "    #{klass}.new(options.merge(opts))\n" \
-          "  end\n" \
-          "  @root = Elem.new(name, proc, options[:content])\n" \
-          'end'
-        )
+        singleton_class.class_eval <<-EVAL, __FILE__, __LINE__ + 1
+          def #{accel_name}(name, options={}) # def sequence(name, type, options)
```

Comment:
Rubocop required the static example to be added as an inline comment:

https://github.com/sdaubert/rasn1/pull/18#discussion_r999801887